src/client.ts

```diff
@@ -113,7 +114,9 @@ export default class Client {
     });
 
     this.eventBus.on(StreamEvent.DISCONNECTED, () => {
-      this.pollProcessor.start();
+      if (!this.closing) {
+        this.pollProcessor.start();
+      }
```

Comment:
Isn't it crazy how many times a bug can come down to a single missed check? Nice find and fix 👍🏻